StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/FinancialConnectionsAPIClient.swift

```diff
@@ -8,8 +8,54 @@
 import Foundation
 @_spi(STP) import StripeCore
 
-protocol FinancialConnectionsAPIClient {
+class FinancialConnectionsAPIClient {
```

Comment:
`FinancialConnectionsAPIClient` is likely a good name because it matches the precedent of `STPAPIClient` 👍

Reply:
could add `final` next to `class` here too for extra clarity